dts/arm/gigadevice/gd32f4xx/gd32f470.dtsi

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022, Rtone.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gigadevice/gd32f4xx/gd32f4xx.dtsi>
+
+/ {
+
```

Comment:
note: these changes are part of the board commit, but they are not strictly related (like SoC changes), please split into its own commit

Reply:
done.